specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -1331,11 +1332,7 @@
           "description": "Gets or sets the etag of the resource."
         }
       },
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
+      "x-ms-azure-resource": true,
```

Comment:
Why has this `allOf` been removed?

Reply:
This resource is a ProxyResource and does not have all the requirements of the Resource (like Location, Tags etc.). Thus I removed the allOf - Resource and just added the tag.

To make it better, I can go and take this route as mentioned here - https://github.com/Azure/azure-rest-api-specs/pull/2368#discussion_r164938982. By creating a separate Proxy Resource.